### PR TITLE
CSS bundling - Ruby on Rails Course

### PR DESCRIPTION
**Pull request by odilsoncode**
## Because

Because the link below shows that the bootstrap install file has been updated and count now 32 lines.
Screenshot 2024-01-05 at 20-38-53 CSS Bundling The Odin Project

https://github.com/rails/cssbundling-rails/blob/main/lib/install/bootstrap/install.rb

## This PR

- Update the build script 

## Issue

Closes #XXXXX

## Additional Information

No more information

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: ruby_on_rails/advanced_forms_and_activerecord/active_record_queries.md [hunk 1]
```diff
@@ -220,8 +220,8 @@ Sometimes, you just can't get ActiveRecord to do what you want it to. In that ca
 
 1. Read the first 6 sections of the [Rails Guide on Active Record Querying](http://guides.rubyonrails.org/active_record_querying.html) for a more basic overview of query functions. Don't worry too much about batching and `#find_each`.
 2. Read section 20 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#existence-of-objects) for a brief look at using `exists?`, `any?` and `many?`.
-3. Read sections 7, 8 and 21 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#group) for an understanding of aggregate functions and the calculations you can run on them.
-4. Skim sections 9-12 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#overriding-conditions).
+3. Read sections 7, 8 and 21 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#grouping) for an understanding of aggregate functions and the calculations you can run on them.
+4. Skim sections 9-12 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#overriding-conditions).
```
Comment (JoshDevHub) [inline ruby_on_rails/advanced_forms_and_activerecord/active_record_queries.md:224]: Why are these changes here?

File: ruby_on_rails/rails_sprinkles/css_bundling.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ That's all there is to it.
...
 
-It's interesting to see how simple this gem is. If you see the [lib/install](https://github.com/rails/cssbundling-rails/tree/main/lib/install) directory for the gem you'll notice how few files there are. The `install.rb` file handles making sure all the necessary parts are in place to handle whichever install option you use, and then each tool has its own directory with the files required for that tool to work. Have a look at the [Bootstrap](https://github.com/rails/cssbundling-rails/blob/main/lib/install/bootstrap/install.rb) install file. It's only 27 lines long.
+It's interesting to see how simple this gem is. If you see the [lib/install](https://github.com/rails/cssbundling-rails/tree/main/lib/install) directory for the gem you'll notice how few files there are. The `install.rb` file handles making sure all the necessary parts are in place to handle whichever install option you use, and then each tool has its own directory with the files required for that tool to work. Have a look at the [Bootstrap](https://github.com/rails/cssbundling-rails/blob/main/lib/install/bootstrap/install.rb) install file. It's only 32 lines long.
```
Comment (JoshDevHub) [inline ruby_on_rails/rails_sprinkles/css_bundling.md:59]: I think it's probably not a great idea for the lesson to refer to the specific line length of the file. This will always be subject to change, and any time it does change, what we say in the lesson will be inaccurate.

Maybe this can just be something like:
```suggestion
It's interesting to see how simple this gem is. If you see the [lib/install](https://github.com/rails/cssbundling-rails/tree/main/lib/install) directory for the gem you'll notice how few files there are. The `install.rb` file handles making sure all the necessary parts are in place to handle whichever install option you use, and then each tool has its own directory with the files required for that tool to work. Have a look at the [Bootstrap](https://github.com/rails/cssbundling-rails/blob/main/lib/install/bootstrap/install.rb) install file. Notice how short it is.
```

**Review by JoshDevHub — CHANGES_REQUESTED** [review]
Couple comments. Let me know what you think.

File: ruby_on_rails/advanced_forms_and_activerecord/active_record_queries.md (hunk 1 above)
Comment (odilsoncode) [reply ruby_on_rails/advanced_forms_and_activerecord/active_record_queries.md:224]: By adding #grouping, the link'll point to the specific section where we want the student starts reading instead of letting him looking for the section from the top of the page.

File: ruby_on_rails/rails_sprinkles/css_bundling.md (hunk 2 above)
Comment (odilsoncode) [reply ruby_on_rails/rails_sprinkles/css_bundling.md:59]: Yes, i totally agree with your change

File: ruby_on_rails/advanced_forms_and_activerecord/active_record_queries.md (hunk 1 above)
Comment (JoshDevHub) [reply ruby_on_rails/advanced_forms_and_activerecord/active_record_queries.md:224]: Ah I see.

This looks like you have some changes that are leaking over from #26955 . If that's the case, you should change this file so it reflects what's currently in `main` and leave the changes around these links to that PR instead of this one.

This is just to avoid introducing conflicts between your different PRs (which tend to be a bit annoying to fix).

File: ruby_on_rails/rails_sprinkles/css_bundling.md (hunk 2 above)
Comment (JoshDevHub) [reply ruby_on_rails/rails_sprinkles/css_bundling.md:59]: Awesome. Feel free to make a new commit to this PR that adds that change in. Or you could click the "Commit suggestion" button that appears under my suggested change. Either way is fine with me.

Comment (odilsoncode) [reply ruby_on_rails/rails_sprinkles/css_bundling.md]: Hi mates, sorry for the delay, i've just make a new commit with the apropriate changes.

**Review by JoshDevHub — APPROVED** [review]
Thanks for your contribution
